nevergrad/optimization/base.py

```diff
@@ -398,7 +425,7 @@ def ask(self) -> p.Parameter:
             callback(self)
         current_num_ask = self.num_ask
         # tentatives if a cheap constraint is available
-        MAX_TENTATIVES = 1000
+        MAX_TENTATIVES = self._max_constraints_trials if self._max_constraints_trials else 1000
```

Comment:
just inititialize `self._max_constraints_trials` to 1000 and use it directly in the code (no need for `MAX_TENTATIVES`

Reply:
Definitely right.